config/default.yml

```diff
@@ -394,6 +394,13 @@ RSpec/MultipleExpectations:
   VersionChanged: '1.21'
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/MultipleExpectations
 
+RSpec/MultipleMemoizedHelpers:
+  Description: Checks if example groups contain too many `let` and `subject` calls.
+  Enabled: true
```

Comment:
⚠️ 
I personally don't have many doubts that this cop should be enabled by default. Open for the discussion, though.

Just in case, we only have a handful of disabled cops, and we encourage (not documented) to tune the configuration to meet the project style, not bend all projects to our defaults.

Reply:
I'm fine with having it enabled, with `AllowSubject` true, by default

Reply:
What about `Enabled: pending`? This cop broke my builds.

Reply:
@AlexWayfer We didn't formally agree to adhere to this policy yet, neither we're sure that it works for extensions.
We'll do our best to make it work and to introduce new cops between 2.0 and 3.0 as `pending`.
Meanwhile - please accept my apologies.